src/node_builtins.h

```diff
@@ -66,6 +66,11 @@ class NODE_EXTERN_PRIVATE BuiltinLoader {
   static v8::MaybeLocal<v8::Value> CompileAndCall(
       v8::Local<v8::Context> context, const char* id, Realm* realm);
 
+  static v8::MaybeLocal<v8::Function> LookupAndCompile(
```

Comment:
Can you add a comment here that this is specifically for Electron? Otherwise this just looks like an unused function that could be removed at any point (and I don’t think this is something we want to make an actual public API either.)